cosmos/spring-cloud-azure-starter-cosmos/spring-cloud-azure-cosmos-sample/pom.xml

```diff
@@ -17,6 +17,11 @@
   <name>Read and Write Data with Azure Cosmos DB Using CosmosClient</name>
   <description>Sample project for Spring Cloud Azure Starter Cosmos</description>
 
+  <properties>
+    <imageName>spring-cloud-azure-cosmos-sample</imageName>
+    <mainClass>com.azure.spring.sample.cosmos.CosmosSampleApplication</mainClass>
+  </properties>
+
```

Comment:
why do we need this?

Reply:
I got the `command too long` exception when using native-image command, the solution is to use the shade profile. These two properties are used for the shade maven plugin to define the main class, if the user uses the native profile is OK then can ignore the shade profile and the properties.